### DISCUSSION TITLE
Renogy Bluetooth Support in future?

### DISCUSSION THREAD

**Discussion by TimGFoley** [Louisvdw/dbus-serialbattery#286] [Q&A]
For some reason Renogy appears to be switching all their newer batteries to Bluetooth comms and removed the RS485 connector.  Has anyone experimented with this yet?

I know for Victron you need to add an external USB Bluetooth dongle for reliable comms since they disable the internal one if the cpu gets warm.

**Comment by mr-manuel** [comment]
In the meanwhile there is a branch `jkbms_ble` that adopts Bluetooth connection to JKBMS. Maybe this helps you a bit to create your own Bluetooth driver for Renogy.
